### ISSUE TITLE
Visual Studio errors

### ISSUE THREAD

**Issue by SomeDustil** [retreev/PangLib.PSP#2]
I wasn't actually sure what to put for the title, but basically, I am COMPLETELY inept with this stuff, and just wanted to mess around with the files for golf.

I ran the dotnet commands, made a reference to the psp pak library, and edited the .cs file param lines to the proper paths (was I even supposed to do this??) with notepad++.

But I have no idea if I did it properly, or if I'm supposed to do other things, because when I try building it in Visual Studios (I also don't know if I'm supposed to use this) or compiling it, I keep getting errors like "A project with an Output Type of Class Library cannot be started directly.", or when I try using csc in the command line, "A namespace cannot directly contain members such as fields or methods."

tl;dr: I know nothing, is there a guide for newbies anywhere??

**Comment by pixeldesu** [comment]
Sorry for the late response (I just didn't look into this project specifically, whoops).

Anyway, Visual Studio/C# is right here, you cannot start a library project directly (it basically just outputs a `.dll` file). You at least want to create a Console (CLI) application.

This one can be properly debugged and executed and that should yield better results.
